realm-annotations-processor/src/test/java/io/realm/processor/RealmProcessorTest.java

```diff
@@ -106,7 +108,7 @@ public void compareProcessedAllTypesFile() throws Exception {
                 .processedWith(new RealmProcessor())
                 .compilesWithoutError()
                 .and()
-                .generatesSources(allTypesProxy);
+                .generatesSources(allTypesDefaultMediator, allTypesDefaultModule, allTypesDefaultMediator);
```

Comment:
Hmm, thought that was done elsewhere. You are right, big mistake, will readd it.